README.md

```diff
@@ -152,7 +152,7 @@ public class UserServiceImpl implements UserService {
 Don't forget that your code will need to be tested, so try to anticipate all invalid input data and handle it in advance.
 For example:
 1. Incorrect file path
-2. Incorrect data in the input file, for example, quantity less than zero or incorrect strategy
```

Comment:
It's not a problem, but it seems that your IDE changed the text in Readme for some reason)